templates/config/i18n-tasks.yml

```diff
@@ -94,7 +95,7 @@ search:
   ##     User.human_attribute_name(:email) and User.model_name.human
   ##
   ## To implement your own, please see `I18n::Tasks::Scanners::AstMatchers::BaseMatcher`.
-  <%# I18n::Tasks.add_ast_matcher('I18n::Tasks::Scanners::AstMatchers::RailsModelMatcher') %>
+  # <%# I18n::Tasks.add_ast_matcher('I18n::Tasks::Scanners::AstMatchers::RailsModelMatcher') %>
```

Comment:
I'm not sure how this makes commenting out easier?

Reply:
1. First of all, I think it is easier to visually understand that *this block* is commented out if it is also commented out as YAML like this.
2. The other parts are also in the format of eRuby-style comments + YAML-style comments ([here](https://github.com/glebm/i18n-tasks/blob/deba8884a20b05e4df65ca99ab3a5f4887815a47/templates/config/i18n-tasks.yml#L34-L35) or [here](https://github.com/glebm/i18n-tasks/blob/deba8884a20b05e4df65ca99ab3a5f4887815a47/templates/config/i18n-tasks.yml#L142-L153) etc), so there is also consistency.
3. I think most people use the comment in/out feature or plugin in their favorite editors. In that case, I think it's possible (and I've got it to happen) that when you try to comment this block in on a block-by-block basis, it gets commented out the other way around.